.github/workflows/release.yml

```diff
@@ -156,7 +156,7 @@ jobs:
           done
 
           mv ./target/debug/sui${{ env.extention }} ${{ env.TMP_BUILD_DIR }}/sui-debug${{ env.extention }}
-          mv ./external-crates/move/target/release/move-analyzer${{ env.extention }} ${{ env.TMP_BUILD_DIR }}/move-analyzer${{ env.extention }}
+          mv ./target/release/move-analyzer${{ env.extention }} ${{ env.TMP_BUILD_DIR }}/move-analyzer${{ env.extention }}
```

Comment:
same here, I think we can just remove this line if it builds as part of `cargo build --release`

Reply:
I think all we have to do is just to add the file to https://github.com/MystenLabs/sui/blob/main/binary-build-list.json and it will get packaged automatically, just like the rest of them

Reply:
Thank you for pointing that out! I followed the advice, but added `move-analyzer` there as I believe it should be the name of the binary rather than the build target (from what I see the `exe` extension is added for Windows build so we should indeed be good there)